src/plugins/intel_cpu/src/utils/shape_inference/shape_inference_status.hpp

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+namespace ov {
+namespace intel_cpu {
+
+enum class ShapeInferStatus {
+    update, // shapes were successfully calculated
```

Comment:
`update` -> `success` as shape `infer` return value not update it?

Reply:
Changed, thank you!